index.html

```diff
@@ -0,0 +1,71 @@
+<html>
+
+<head>
+    <title>Nikos Village Websitelol</title>
+</head>
+
+<body>
+    
+    <div class="FadeIn"><h1>Welcome to Drosia!</h1></div>
+  
+    <div class="Fadein"><p>This is the test paragraph for the site. Maybe one day
+        it wont look like it was made in 1999.
+    </p></div>
+
+    <div  class="FadeIn"><button>Test Button</button></div>
+    <script src="Thing.js"></script>
```

Comment:
Script tags should always be at the very end of the `<body>` tags, this is because if your javascript is attempting to find something on the page (like a button), that button needs to exist _before_ the javascript can find it. 

Hence, always put script tags at the very end of the body.

Reply:
@NikoTheGreek1821 it looks like this script tag is still here, it should be moved to the bottom of the body.